qute.jdt/com.redhat.qute.jdt.test/src/main/java/com/redhat/qute/jdt/template/TemplateGenerateMissingJavaMemberTest.java

```diff
@@ -187,7 +187,7 @@ public void generateTemplateExtensionInExistingClass() throws Exception {
 		IJavaProject project = loadMavenProject(QuteMavenProjectName.qute_quickstart);
 		GenerateMissingJavaMemberParams params = new GenerateMissingJavaMemberParams(
 				GenerateMissingJavaMemberParams.MemberType.AppendTemplateExtension, "asdf", "org.acme.qute.Item",
-				QuteMavenProjectName.qute_quickstart);
+				QuteMavenProjectName.qute_quickstart, "org.acme.qute.MyTemplateExtensions");
```

Comment:
Defines this template class in private static field. In the future perhaps it should be nice to have settings for that. @fbricon what do you think about that?

Reply:
Just FYI this code is for a test where code is being added to an existing class that's annotated with `@TemplateExtension`. The code to generate a new `@TemplateExtension` class is: https://github.com/redhat-developer/quarkus-ls/blob/8a52980f2a999e80a0367f301f8962dfaf4f84b7/qute.jdt/com.redhat.qute.jdt/src/main/java/com/redhat/qute/jdt/internal/template/QuteSupportForTemplateGenerateMissingJavaMemberHandler.java#L466

Reply:
Ooops sorry wrong location code. Ok let's keep like this but I think we should perhaps a settings for that. Please create an issue.

Reply:
https://github.com/redhat-developer/quarkus-ls/issues/699